src/ui_utils.h

```diff
@@ -63,7 +63,7 @@ typedef struct GeanyInterfacePrefs
 	gboolean		msgwin_compiler_visible;	/**< whether message window's compiler tab is visible */
 	gboolean		msgwin_messages_visible;	/**< whether message window's messages tab is visible */
 	gboolean		msgwin_scribble_visible;	/**< whether message window's scribble tab is visible */
-	/** whether to use native Windows' dialogs (only used on Windows) */
+	/** whether to use native Windows' dialogs - ignored and not used anymore */
```

Comment:
Since this field is part of the API (and ABI), we cannot easily remove it.

Is it enough to mark it as ignored in the comment or are there better ways to handle this?

Reply:
I would suspect that there are not many Windows specific plugins out there, so probably ok.

Reply:
Actually so long as it is still initialized to say "Windows dialogs not used" then its definitely ok.

Reply:
It's not initialized explicitly at all. But at least on my Debian Testing, the resulting value when Geany is running is `FALSE`.